docs/tutorials/scenario-simulation/planning-simulation/random-test-simulation.md

```diff
@@ -0,0 +1,28 @@
+# Random test simulation
+
+!!! note
+
+    Running the Scenario Simulator requires some additional steps on top of building and installing Autoware, so make sure that [Scenario Simulator installation](installation.md) has been completed first before proceeding.
+
+## Running steps
+
+1. Move to the workspace directory where Autoware and the Scenario Simulator have been built.
+
+2. Source the workspace setup script:
+
+   ```bash
+   source install/setup.bash
+   ```
+
+3. Run the simulation:
+
+   ```bash
+   ros2 launch random_test_runner random_test.launch.py \
+     architecture_type:=awf/universe \
+     sensor_model:=sample_sensor_kit \
+     vehicle_model:=sample_vehicle
+   ```
+
+![random_test_runner](images/random_test_runner.png)
+
+For more information about supported parameters, refer to the ![random_test_runner documentation].(<https://github.com/tier4/scenario_simulator_v2/blob/master/test_runner/random_test_runner/Readme.md#node-parameters>)
```

Comment:
@WJaworskiRobotec cc @HansRobo Could you update this link? It's 404 now.
https://github.com/autowarefoundation/autoware-documentation/runs/6769693334?check_suite_focus=true#step:3:581

Reply:
Thank you for your kind notification!
I updated the link.
```suggestion
For more information about supported parameters, refer to the ![random_test_runner documentation].(<https://github.com/tier4/scenario_simulator_v2/blob/master/docs/user_guide/random_test_runner/README.md#node-parameters>)
```

Reply:
@HansRobo Could you open a PR for that? :pray:

Reply:
Or should I do it?

Reply:
Oh, This PR has already merged!
I will create PR for this change now...